Source/ACE/Entity/MovementData.cs

```diff
@@ -250,6 +169,29 @@ public MovementData ConvertToClientAccepted(uint holdKey, CreatureSkill run)
             return md;
         }
 
+        public void SetMovementStateFlag()
+        {
```

Comment:
I'm going to be nit-picky here -- I don't like having to remember/know to call this function before we get the MovmentStateFlag.  Can we make the accessor (get) of MovementStateFlag do this computation?